docs/release/trg-0/trg-4-08.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TRG 4.08 - Multi-platform
+---
+
+| Status | Created     | Post-History                              |
+|--------|-------------|-------------------------------------------|
+| Draft  | 14-Mar-2024 | Initial contribution                      |
+
+## Why
+
+Providing container images that support multiple CPU architectures enhances flexibility, compatibility, and efficiency across hardware devices.
+
+This is especially beneficial for supporting use cases like tutorials and local playgrounds.
+
+## Description
+
+Container images **must** be built for **amd64 and arm64** CPU architectures.
+
+### Implementation
+
+Add and enhance the GitHub Workflow with the following steps:
+
+```yaml
+name: Build - Docker image (SemVer)
+
+...
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      ...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up QEMU
```

Comment:
will this step also a must and is it nessesary for multi-build images? 
https://github.com/qemu/qemu

Or just additional information about the emulation and virtualisation,  then i would integrate a little bit more information on the "Why" section why we also want to have this step in the workflow files.

Reply:
I added a comment that those steps are necessary for multi-platfrom images